src/js/utils.js

```diff
@@ -118,3 +89,39 @@ export function getModifier(obj, key) {
 export function includes(a, b) {
   return a.indexOf(b) > -1
 }
+
+/**
+ * Determines if the value is singular-like
+ * @param {any} value
+ * @return {Boolean}
+ */
+export function isSingular(value) {
+  return isPlainObject(value) || value instanceof Element
+}
+
+/**
+ * Tricking bundlers, linters, and minifiers
+ * @return {String}
+ */
+export function innerHTML() {
+  return 'innerHTML'
+}
```

Comment:
I'm still not sure why this is needed

Reply:
It's because Firefox extensions (maybe other things) don't allow you to set `.innerHTML` for security reasons, so we "trick" it with this technique so it can't tell we're setting the innerHTML, lol

Reply:
Sounds like perfect "security" implementation 😄

Reply:
I can't figure out how to trick minifiers (terser is smart enough to reduce it to .innerHTML) but for extensions people can use the unminified version